index.html

```diff
@@ -142,53 +183,58 @@ <h2 class="about-sub-head">Let's Connect</h2>
                         <li><a href="#"><img src="icon/git.svg"></a></li>
                     </ul>
                 </div>
-                     <button class="resume-btn">Get My resume</button>
+                   <button class="resume-btn">Get My resume</button>
             </div>
+
+            
             <div class="language-header">
-                <h3 class="language-heading">Languages<i class="fa fa-angle-down"></i></i></h3>
-              </div>
-              <ul class="list-language">
-                  <li class="for-web"><img src="icon/js.png" alt="">JavaSript</li>
-                  <li class="for-web"><img src="icon/html.png" alt="">HTML</li>
-                  <li class="for-web"><img src="icon/css.png" alt="">CSS</li>
-              </ul> 
-              <ul class="frame-work">
-                  <li class="skill">
-                      <h3 class="first-title">Frameworks<i class="fa fa-angle-right" ></i></h3>
-                      <hr class="line">  
-                  </li>
-                  <li class="skills">
-                      <h3 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h3> 
-                      <hr class="line">
-                  </li>
-              </ul>
-             </section>
 
-             <section class="form">
+                <h3 class="language-heading">Languages <img src="images/down.png" alt="" srcset=""></i></h3>
+
+            <ul id="for-desk" class="list-language">
+                <li class="for-web"><img src="icon/js.png" alt=""><p>JavaSript</p></li>
+                <li class="for-web"><img src="icon/html.png" alt=""><p>HTML</p></li>
+                <li class="for-web"><img src="icon/css.png" alt=""><p>CSS</p></li>
+            </ul> 
+            <ul class="frame-work">
+                <li class="skill">
+                    <h3 id="desktop1" class="first-title">Framework <img src="images/Union.png" alt=""></i></h3>
+                    <hr class="line">  
+                </li>
+                <li class="skills">
+                    <h3 id="desktop2" class="second-title">Skills<img src="images/Union.png" alt=""></i></h3> 
+                    <hr class="line">
+                </li>
```

Comment:
- [ ] As the requirement for this milestone is to create the desktop version of your portfolio, Your contact form looks great on the desktop, and you also implemented the suggestions from the previous reviewer, but it seems like some CSS you introduced makes your mobile version deviate from the design, 👍 Please you need to fix this as your final portfolio is meant to look great both in desktop and mobile. 😄 
- - Your current design:
![Screenshot 2023-04-01 205807](https://user-images.githubusercontent.com/107632780/229311974-e249955a-7f35-4dc2-902c-8847427948a3.png)
- - Figma design:
![Screenshot 2023-04-01 205710](https://user-images.githubusercontent.com/107632780/229312020-cb5c8c28-50d2-48d1-be1a-370f61f98c27.png)